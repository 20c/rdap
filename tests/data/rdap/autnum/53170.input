{"objectClassName":"autnum","handle":"53170","startAutnum":53170,"endAutnum":53170,"name":"ASN53170","type":"DIRECT ALLOCATION","country":"BR","links":[{"value":"https://rdap.registro.br/autnum/53170","rel":"self","href":"https://rdap.registro.br/autnum/53170","type":"application/rdap+json"},{"value":"https://rdap.registro.br/autnum/53170","rel":"related","href":"https://rdap.registro.br/ip/186.224.48.0/20","type":"application/rdap+json"},{"value":"https://rdap.registro.br/autnum/53170","rel":"related","href":"https://rdap.registro.br/ip/2804:3838::/32","type":"application/rdap+json"}],"entities":[{"objectClassName":"entity","handle":"17769837000100","vcardArray":["vcard",[["version",{},"text","4.0"],["kind",{},"text","org"],["fn",{},"text","Cabangu Internet Ltda"],["adr",{},"text",["","","","","","","BR"]]]],"roles":["registrant"],"publicIds":[{"type":"cnpj","identifier":"17.769.837/0001-00"}],"entities":[{"objectClassName":"entity","handle":"GJM3","vcardArray":["vcard",[["version",{},"text","4.0"],["kind",{},"text","individual"],["fn",{},"text","Geovane Jose Vieira Martins"],["email",{},"text","cpd@cabanguinternet.com.br"],["lang",{},"language-tag","PT"],["adr",{},"text",["","","","","","","BR"]]]],"roles":["administrative"],"events":[{"eventAction":"registration","eventDate":"1998-11-10T12:00:00Z"},{"eventAction":"last changed","eventDate":"2016-04-26T11:58:32Z"}],"links":[{"value":"https://rdap.registro.br/entity/GJM3","rel":"self","href":"https://rdap.registro.br/entity/GJM3","type":"application/rdap+json"}]}],"events":[{"eventAction":"registration","eventDate":"1998-11-10T12:00:00Z"},{"eventAction":"last changed","eventDate":"2011-08-11T20:18:33Z"}],"links":[{"value":"https://rdap.registro.br/entity/17769837000100","rel":"self","href":"https://rdap.registro.br/entity/17769837000100","type":"application/rdap+json"}],"legalRepresentative":"Geovane Jose Vieira Martins"},{"objectClassName":"entity","handle":"GJM3","vcardArray":["vcard",[["version",{},"text","4.0"],["kind",{},"text","individual"],["fn",{},"text","Geovane Jose Vieira Martins"],["email",{},"text","cpd@cabanguinternet.com.br"],["lang",{},"language-tag","PT"],["adr",{},"text",["","","","","","","BR"]]]],"roles":["administrative","abuse"],"events":[{"eventAction":"registration","eventDate":"1998-11-10T12:00:00Z"},{"eventAction":"last changed","eventDate":"2016-04-26T11:58:32Z"}],"links":[{"value":"https://rdap.registro.br/entity/GJM3","rel":"self","href":"https://rdap.registro.br/entity/GJM3","type":"application/rdap+json"}]}],"events":[{"eventAction":"registration","eventDate":"2010-07-26T14:27:24Z"},{"eventAction":"last changed","eventDate":"2013-03-06T14:00:54Z"}],"notices":[{"title":"Terms of Use","description":["Copyright (c) NIC.br","The use of the data is only permitted as described in","full by the terms of use, being prohibited its ","distribution, commercialization or reproduction, in ","particular, to use it for advertising or any similar ","purpose."],"links":[{"value":"https://rdap.registro.br/autnum/53170","rel":"alternate","href":"https://registro.br/termo/en.html","type":"text/html"}]}],"rdapConformance":["rdap_level_0","nicbr_level_0"],"port43":"whois.nic.br"}
