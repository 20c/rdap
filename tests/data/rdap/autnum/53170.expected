{
  "name": "ASN53170",
  "emails": [],
  "org_name": "Cabangu Internet Ltda",
  "org_address": ""
}